app/src/main/res/values/strings.xml

```diff
@@ -14,6 +14,8 @@
     <string name="search_hint">Search...</string>
     <string name="search_go_button">Go</string>
     <string name="latitude">Latitude</string>
+    <string name="sell">Sell</string>
+    <string name="unsell">Unsell</string>
```

Comment:
I think the words "Sell" and "Unsell" is a bit weird in this context, as upon clicking the button in status "Sell" will not really sell the item right away. For example, I think it would be more user-friendly to use a "Put on sale" or "List for sale" text which describes what the button is doing when the user clicks it, same for the "Unsale" text, something like "Remove from sale" would be a bit more understandable in my opinion.

Reply:
Indeed, but maybe we could find something shorter?

Reply:
I looked up synonyms but nothing was explicit/understandable enough :(, do you have any ideas?

Reply:
Watch this:
```suggestion
    <string name="unsell">Resell</string>
```

Reply:
Look ma, I didn't even open Android Studio